app/src/main/java/app/revanced/manager/ui/screen/settings/ImportExportSettingsScreen.kt

```diff
@@ -59,6 +65,8 @@ fun ImportExportSettingsScreen(
     vm: ImportExportViewModel = koinViewModel()
 ) {
     val context = LocalContext.current
+    val coroutineScope = rememberCoroutineScope()
+    var selectorDialog by remember { mutableStateOf<(@Composable () -> Unit)?>(null) }
```

Comment:
This state will be lost on screen rotation since its not in a viewmodel or `rememberSaveable`. Is this intentional?